ui/app/AppLayouts/Communities/popups/CreateChannelPopup.qml

```diff
@@ -631,6 +633,12 @@ StatusStackModal {
                     ]
                 }
             }
+        },
+        ColorPanel {
```

Comment:
This shouldn't be a regular page (part of the flow); instead it should be load/replaced and then it would automagically handle the buttons (back, rightButtons)